### ISSUE TITLE
Can't complete install on Windows 7

### ISSUE THREAD

**Issue by SnGrimes** [Automattic/wp-calypso#25187]
#### Steps to reproduce
1. I used git clone to download repository to my machine
2. I made sure to have node 8.11.0 and npm 5.6.0
3. run` npm start`
4. get error that 'CALYPSO_SERVER' is not recognized as an internal or external command

#### What I expected
wp-calypso would install and I could run it on 127.0.01 calypso.localhost

#### What happened instead
I get this error while npm is trying to set up calypso

> 'CALYPSO_SERVER' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wp-calypso@0.17.0 start: `npm run -s build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the wp-calypso@0.17.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

#### Browser / OS version
Windows 7 Home Premium SP1

#### Context / Source
I would like to contribute to the calypso project but I this is stopping from getting started.

**Comment by nylen** [comment]
Hi @SnGrimes, thanks for the report.  It looks like there are some problems running the Calypso build on Windows.

I've created a branch and a pull request (https://github.com/Automattic/wp-calypso/pull/25188) to try to address some of the issues I see.  I'm not using Windows though, so I'll need your help testing.  Can you try running again on my branch?

```
git fetch origin
git checkout origin/fix/build-on-windows -B fix/build-on-windows
npm start
```

**Comment by SnGrimes** [comment]
I was able to install calypso using your branch and it is loading on http://calypso.localhost:3000/ with the Wordpress logo in the center. However, there are some 404 errors: 
![image](https://user-images.githubusercontent.com/14166723/40744810-508f7abe-641b-11e8-84eb-07ac19050822.png)

**Comment by nylen** [comment]
Ok - that's good progress, but it looks like something is still not working correctly.  Can you see any other errors further up in the build process?

**Comment by SnGrimes** [comment]
I've scanned the terminal and the only thing I could find was that the npm update check failed due to lack of access. 
![image](https://user-images.githubusercontent.com/14166723/40746639-9454ced4-6420-11e8-93fb-332ee9be5e6b.png)
This happened before the webpack build though. There were a couple deprecation warnings: 
![image](https://user-images.githubusercontent.com/14166723/40746662-a6c09b52-6420-11e8-9bbb-198813b5624e.png)
Everything is green and [emitted] ...
I opened the dev tools and there was a TypeError if that helps. 
![image](https://user-images.githubusercontent.com/14166723/40746828-139612ac-6421-11e8-8db2-9c1697a13ff4.png)

**Comment by nylen** [comment]
All of these built files like `/calypso/manifest.js` should be served from the `public/` directory in your Calypso repository.

Is http://calypso.localhost:3000/calypso/manifest.js the URL that your browser is trying to load for `manifest.js`?

Does `public/manifest.js` exist in your repository?

**Comment by nylen** [comment]
Relevant code:

https://github.com/Automattic/wp-calypso/blob/2d3ad83a/webpack.config.js#L78-L80

https://github.com/Automattic/wp-calypso/blob/2d3ad83a/server/boot/index.js#L69-L70

**Comment by SnGrimes** [comment]
Yes, the browser is trying to load from that URL. `manifest.js` is missing from the public folder. There is only a `manifest.json`.

**Comment by nylen** [comment]
That is pretty strange - it means that the Webpack build appears to succeed but it is failing to generate its output files.

If you run `npm run build-client` - does the same thing happen?

I think I am nearing the limit of what I can do to help here.  I will try to get someone who runs Windows to take a look and help you out.

**Comment by SnGrimes** [comment]
That did it! You're amazing! 
![image](https://user-images.githubusercontent.com/14166723/40751152-9ec1a022-642f-11e8-8c60-2a364a606bd9.png)
There aren't any errors in the console or terminal.

**Comment by nylen** [comment]
Right, because my pull request that fixes things hasn't been merged to `master` yet.  I'll do that now.

**Comment by SnGrimes** [comment]
This issue was fixed by #25188. I pulled the new master and `npm start` ran without issues and I can open calypso.
